Docs/ProjectSpec.md

```diff
@@ -130,6 +132,81 @@ options:
     tvOS: "10.0"
 ```
 
+### Breakpoints
+
+- [x] **type**: **String** - Breakpoint type
+    - `FileBreakpoint`: file brakpoint
+    - `ExceptionBreakpoint`: exception breakpoint
+    - `SwiftErrorBreakpoint`: swift error breakpoint
+    - `OpenGLErrorBreakpoint`: OpenGL breakpoint
+    - `SymbolicBreakpoint`: symbolic breakpoint
+    - `IDEConstraintErrorBreakpoint`: IDE constraint breakpoint
+    - `IDETestFailureBreakpoint`: IDE test failure breakpoint
+- [ ] **enabled**: **Bool** - indicates whether it should be active
+- [ ] **ignoreCount**: **String** - indicates how many times it should be ignored before stopping
```

Comment:
While all of these properties are currently strings in `XcodeProj` we should handle this better and make the relevant ones ints like:
- ignoreCount
- startingColumn
- endingColumn
- startingLine
- endingLine

The BreakpointGenerator can then turn these into strings for now